feature/player/src/main/java/dev/anilbeesetti/nextplayer/feature/player/PlayerActivity.kt

```diff
@@ -289,6 +300,44 @@ class PlayerActivity : AppCompatActivity() {
         super.onStop()
     }
 
+    @RequiresApi(Build.VERSION_CODES.O)
+    override fun onUserLeaveHint() {
+        super.onUserLeaveHint()
+        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.R && playerPreferences.autoPip) {
```

Comment:
What's the need to verify version R? Pip is compatible starting from version O

Reply:
For Android > R, SetAutoEnabled will do the job with better animation hence manual calling enterPipOnUserLeaveHint is not required.